target/linux/qualcommax/ipq807x/base-files/lib/upgrade/platform.sh

```diff
@@ -90,6 +90,7 @@ platform_do_upgrade() {
 		fw_setenv auto_recovery yes
 		nand_do_upgrade "$1"
 		;;
+	spectrum,sax1v1k|\
```

Comment:
P Q R S, it's in the wrong place needs to be below qnap

Reply:
Sorry, case sensitive from here as well.

Reply:
fixed